src/js/repromise.re

```diff
@@ -21,23 +21,24 @@ function unwrap(value) {
         return value;
 }
 
+function wrap(value) {
```

Comment:
This is cleaner, but does result in an additional function call.

Reply:
I'd left this as is because I was unsure whether these wrappers were being optimized for perf.